src/components/Card.tsx

```diff
@@ -1,24 +1,84 @@
 import type { ReactNode } from 'react';
 import React from 'react';
-import { StyleProp, View, ViewStyle } from 'react-native';
+import { StyleSheet, TouchableOpacity, View, ViewStyle } from 'react-native';
+
 import { useTheme } from '../styles/themes';
+import { Icon } from './icons/Icon';
+
+import { Body } from './typography/Body';
 
 interface Props {
     children?: ReactNode;
+    bottomChildren: ReactNode;
+    title: string;
+    titleColor: string;
     style?: ViewStyle;
+    disabled?: boolean;
+    withClose?: boolean;
+    onClose?: () => void;
```

Comment:
those two attributes will always be used together ? I find both a little redundant and Im wondering if the boolean is very useful. Cant we just check if onClose callback is set to determine if we should display the close icon ? Maybe Im missing a usecase or something in the design system.

Reply:
Yes, i think it will better to check ony for a callback. Thanks